client.py

```diff
@@ -13,15 +13,29 @@
 my_user = structure.User_obj('Funing','456',chats=[]) # need to modify after demo, maybe change to a string
 my_chat_room = ''
 
-def add_user_client():
+def user_login_client():
+   #maybe use a while loop? while loggedin = false,
+   logged_in = False
+   while logged_in == False:
+      username,password = ("To log in, please enter your username,password").split(",")
+      clientSocket.send(username + "," + password + "," + "user_login_client")
+      message = clientSocket.recv(1024)
+      print(('From Server:'), message) #depending on message, toggle loggedin to be true
+      if message == "match": 
+         logged_in = True
+      elif message == "nomatch": 
+         print("Sorry, looks like you aren't in the system.")
```

Comment:
Should we also have a function that adds users to the system? Bc add_user_client() just adds them to the group chat.